src/lib.rs

```diff
@@ -18,10 +18,10 @@ fn create_table<T: PartialEq + Clone>(old: &[T], new: &[T]) -> Vec<Vec<u32>> {
     let new_len = new.len();
     let old_len = old.len();
     let mut table = vec![vec![0; old_len + 1]; new_len + 1];
-    for (i, _) in new.iter().enumerate() {
+    for i in 0..new_len {
         let i = new_len - i - 1;
         table[i][old_len] = 0;
-        for (j, _) in old.iter().enumerate() {
+        for j in 0..old_len { {
```

Comment:
@mark-summerfield Thanks for your PR. Could you please check duplicate `{` , if you get a chance.